lib/build.gradle.kts

```diff
@@ -39,6 +39,8 @@ tasks.withType<DokkaTask>().configureEach {
             includes.from("lib.md")
         }
     }
+
+    moduleName.set("starknet-jvm")
```

Comment:
Setting this parameter causes for the `lib.md` to not be included as part of docs for some reason.